### PR TITLE
User service UI

### PR DESCRIPTION
**Pull request by wilsonngja**
Implemented User Service UI to see the entry.
There seems to be an issue where server does not whitelist localhost hence, I've also added it in this PR.
Please help to check if any bug arises and if there is, please help to raise the issue.

(30/09 00:20) Edit: Change the number of user (from 1 to 0) that triggers the fetch user error. The change is due to the deletion when 1 user is left. In that scenario, the user will get deleted and can't be fetch. When there is 0 user, the error will appear in the console.

### CODE REVIEW

**Comment by yuxinwu99** [comment]
Looks good to me.

File: backend/user_service/src/usermanager.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ const getUserById = (request, response) => {
     pool.query(query, (error, results) => {
         if (error) {
             const msg = {'msg': error.message, 'user': null};
-            return response.status(500).json(msg);
+            return response.status(500).json(msg);``
```
Comment (choonyongchan) [inline backend/user_service/src/usermanager.js:117]: TIny change: Perhaps we can remove the ``

File: backend/user_service/src/usermanager.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ const getUsers = (request, response) => {
         }
         const allUsers = results.rows;
         const numUsers = allUsers.length;
-        if (numUsers < 1) {
+        if (numUsers < 0) {
```
Comment (choonyongchan) [inline backend/user_service/src/usermanager.js:95]: Interesting edge case to catch. Perhaps we can explore this offline :)

**Review by choonyongchan — APPROVED** [review]
LGTM. Discussed offline that the changes will be made in a separate PR

**Pull request merged**